crawler.py

```diff
@@ -301,6 +311,10 @@ def start_driver(self):
                                                  'temporary': True})
             time.sleep(2)
 
+            # install a parallel extension to run alongside pb
+            if self.load_extension:
+                self.driver.install_addon(self.load_extension)
```

Comment:
Firefox apparently requires absolute paths (while Chrome doesn't). Could we enforce an absolute path here for Firefox? (Providing either an absolute or a relative path to `--load-extension` should work in both browsers, on Docker and without.)

Reply:
I'm not sure what you mean. If the user passes along the argument `parallel-extensions/some_extension.xpi` like the option expects, wouldn't that be the absolute path in both the Docker build context and when running the python script directly?

Reply:
`parallel-extensions/some_extension.xpi` is a relative path.

`/home/daly/badger-sett/parallel-extensions/some_extension.xpi` is an absolute path.

What happens now when you provide a relative path to `--load-extension` on Firefox?

Reply:
I see what you mean, now. I was considering just from the Docker build's point of view. Could be that I just ran through a bunch of updates, but I'm getting the same absolute path error when trying to run a firefox build with no parallel extension option. Just to make sure -- you're able to run a regular Docker build with Firefox?

Reply:
>you're able to run a regular Docker build with Firefox?

Yes.